lib/Doctrine/ORM/Mapping/Driver/AttributeReader.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Mapping\Driver;
+
+use Attribute;
+use Doctrine\ORM\Mapping\Annotation;
+use ReflectionAttribute;
+use ReflectionClass;
+use ReflectionMethod;
+use ReflectionProperty;
+
+use function count;
+use function is_subclass_of;
+
+class AttributeReader
```

Comment:
Why is `Reader` not explicitly implemented here? Is it because `*Annotations` methods are going to be renamed?

Reply:
Because its not supposed to be used with Annotations, its just the same API because I copy adapted the code from AnnotationDriver for AttributesDriver